### PR TITLE
Dyar1234 update

### PR DESCRIPTION
**Pull request by Bibiko**
due to unmatched ElCat.iso

### CODE REVIEW

**Comment by Bibiko** [comment]
sorry for mixing the dirr1240 update PR here (my fault) - but it shouldn't be a problem

File: languoids/tree/afro1255/chad1250/west2785/west2790/west2800/sout3162/sout3170/zaks1240/dyar1234/md.ini [hunk 1]
```diff
@@ -42,4 +43,13 @@ comment = Dyarim (10450-NOCODE_Dyarim) = Severely Endangered (80 percent certain
 [altnames]
 hhbib_lgcode = 
 	Kaiyorawa
+elcat = 
+	Dyarum
+	Dyàrìm Tə́
+	Dyarim Tə
+	Ndyarim Tə
+	Kaiwari
+	Kayaurinci
```
Comment (xflr6) [inline languoids/tree/afro1255/chad1250/west2785/west2790/west2800/sout3162/sout3170/zaks1240/dyar1234/md.ini:52]:
```python
>>> sorted(['Dyarum', 'Dyàrìm Tə́', 'Dyarim Tə', 'Ndyarim Tə', 'Kaiwari', 'Kayaurinci'])
['Dyarim Tə', 'Dyarum', 'Dyàrìm Tə́', 'Kaiwari', 'Kayaurinci', 'Ndyarim Tə']
```

File: languoids/tree/afro1255/chad1250/west2785/west2790/west2800/sout3162/nort3190/nyam1284/polc1243/dirr1240/md.ini [hunk 2]
```diff
@@ -2,17 +2,29 @@
...
 	Baram Dutse
 	Diir
 	Dir
 	Dra
+elcat = 
+	Diir
+	Dììr
+	Diri
+	Dra
+	Baram Dutse
```
Comment (xflr6) [inline languoids/tree/afro1255/chad1250/west2785/west2790/west2800/sout3162/nort3190/nyam1284/polc1243/dirr1240/md.ini:25]:
```python
>>> sorted(['Baram Dutse', 'Diir', 'Diri', 'Dra', 'Dììr'])
['Baram Dutse', 'Diir', 'Diri', 'Dra', 'Dììr']
```

**Comment by Bibiko** [comment]
Closed in favor of a new PR here and in pyglottolog

**Pull request closed without merging**